module-4/backend/week_two.md

```diff
@@ -1,22 +1,71 @@
 ## Week Two - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's one difference between ES5 and ES6?
-2. What's the difference between asynchronous and synchronous JavaScript? 
+
+>ES6 has arrow funciton with implicit returns and ES5 does not
+
+2. What's the difference between asynchronous and synchronous JavaScript?
+
+>Synchronus, operates each function one at a time. Async fires all the functions
```

Comment:
Close Async doesn't exactly fire all the functions at one. It allows functions to be run in the background and handle the results when it is complete.